freestyle-cache/shared/src/main/scala/hashmap/HashMapWrapper.scala

```diff
@@ -41,6 +42,16 @@ final class ConcurrentHashMapWrapper[F[_], Key, Value](
     ()
   }
 
+  override def putAll(keyValues: Map[Key, Value]): F[Unit] = C.capture {
+    keyValues.map { case (k, v) => table.put(hkey(k), v) }
```

Comment:
If we replace `map` here with `foreach`, we don't need the `()` on the next line.